model/inline/src/main/scala/aqua/model/inline/ArrowInliner.scala

```diff
@@ -313,22 +313,28 @@ object ArrowInliner extends Logging {
         case ArrowType(_, _) =>
           Exports
             .getLastValue(currentOldName, exports)
-            .flatMap { case vm @ VarModel(name, _, _) =>
-              arrows
-                .get(name)
-                .map(fa =>
-                  (
-                    Map(currentNewName.getOrElse(currentOldName) -> vm),
-                    Map(name -> fa)
+            .flatMap {
+              case vm @ VarModel(name, _, _) =>
+                arrows
+                  .get(name)
+                  .map(fa =>
+                    (
+                      Map(currentNewName.getOrElse(currentOldName) -> vm),
+                      Map(name -> fa)
+                    )
                   )
-                )
+              case lm @ LiteralModel(_, _) =>
```

Comment:
Why not something like
```
case model =>
  logger.error(s"Unexpected, $model can not be an arrow")
  None
```
?

Reply:
because if there will be another model, we will handle it separately